Common/Product/SharedProject/CommonPackage.cs

```diff
@@ -129,6 +129,10 @@ public abstract class CommonPackage : Package, IOleComponent {
         /// </summary>
         /// <returns></returns>
         public static IWpfTextView GetActiveTextView(System.IServiceProvider serviceProvider) {
+            if (serviceProvider == null) {
+                return null;
+            }
```

Comment:
for both null checks, it's okay to fail gracefully, but let's go ahead and also add a debug assert so we know something's up when the issue occurs.